v2/README.md

```diff
@@ -89,8 +89,17 @@ See the list of supported resources [here](https://azure.github.io/azure-service
         --set azureSubscriptionID=$AZURE_SUBSCRIPTION_ID \
         --set azureTenantID=$AZURE_TENANT_ID \
         --set azureClientID=$AZURE_CLIENT_ID \
-        --set azureClientSecret=$AZURE_CLIENT_SECRET
+        --set azureClientSecret=$AZURE_CLIENT_SECRET \
+        --set crdPattern='resources.azure.com/*;containerservice.azure.com/*;keyvault.azure.com/*;managedidentity.azure.com/*;eventhub.azure.com/*'
    ```
+
+   > **Warning:** Make sure to set the `crdPattern` variable to include the CRDs you are interested in using. 
+   > You can use `--set crdPattern=*` to install all the CRDs, but be aware of the 
+   > [limits of the Kubernetes you are running](https://github.com/Azure/azure-service-operator/issues/2920). `*` is not
+   > recommended on AKS Free-tier clusters.
```

Comment:
```suggestion
   > [limits of the Kubernetes cluster you are running](https://github.com/Azure/azure-service-operator/issues/2920). The pattern `*` is _**not**_
   > recommended on AKS Free-tier clusters.
   
   Are we still calling them free? I thought we were rebranding the tiers.
```

Reply:
I've bolded the not but didn't also underline it (IMO the bolding is sufficient emphasis?)

Reply:
Sometimes I overdo the formatting. 😁 
Looks good.

Reply:
Nit: `_` means _italics_ not underline.